CMakeLists.txt

```diff
@@ -222,11 +222,6 @@ elseif((FREERTOS_PORT STREQUAL "A_CUSTOM_PORT") AND (NOT TARGET freertos_kernel_
     "       freertos_kernel)")
 endif()
 
-########################################################################
-# Requirements
-set(CMAKE_C_STANDARD 99)
```

Comment:
Yes please - you can do this by:

`set_property(TARGET freertos_kernel PROPERTY C_STANDARD 90)`
`set_property(TARGET freertos_kernel_port PROPERTY C_STANDARD 90)`

The reason it was added in the first place was because it wasn't explicitly defined.

Not quite sure why you require a 30 yr old standard, considering most compilers support C99 and above and the customers who aren't willing to change compilers probably aren't willing to consume a newer version of FreeRTOS either.  But that's another discussion.

Reply:
"_Not quite sure why you require a 30 yr old standard, considering most compilers support C99_"

This comes from the experience of trying to provide a single code base that builds with 20 different compilers for 40 different architectures, not all of which are "mainstream" :o)   While those using edge case compilers only make up a small percentage of users, we would rather not break the code for anybody.  There is a new major version of FreeRTOS in the offing, for which we may re-evaluate that decision though.

Reply:
Thank you for your suggestion @phelter. For now, we are adding it for the common kernel source files and not for the port files.